### DISCUSSION TITLE
[Feature] Map Support For Listing Photos

### DISCUSSION THREAD

**Discussion by unnerving-sprinkler** [immich-app/immich#5618] [Feature Request]
### The feature

Love the app, switching over from Synology Photos. 

Would love if when I click on a map location (eg. a bubble that says 30), it opened a album/list of photos as opposed to a single photo and the ability to scroll over to more. (much like the Android app currently does)

Use case: I work at a various locations, and am often looking for a particular photo from one location, a lot easier to scroll through an album then through individual photos. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/4519
